test/unit/spec/Util_spec.js

```diff
@@ -133,6 +135,41 @@ define(['util/Util', 'util/Logger'], function (Util, Logger) {
       });
     });
 
+    describe('postToUrl', function () {
+      beforeEach(function () {
+        spyOn($.fn, 'submit');
+      });
+
+      it('post the from when no query parameters', function () {
```

Comment:
typo `post the form` not the `from`, and maybe `post the form without query parameters`